packages/block-editor/src/components/block-controls/index.js

```diff
@@ -11,24 +11,69 @@ import {
 	__experimentalToolbarContext as ToolbarContext,
 	createSlotFill,
 	ToolbarGroup,
+	__experimentalUseSlot as useSlot,
 } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import useDisplayBlockControls from '../use-display-block-controls';
 
-const { Fill, Slot } = createSlotFill( 'BlockControls' );
+const BlockControlsDefault = createSlotFill( 'BlockControls' );
+const BlockControlsBlock = createSlotFill( 'BlockFormatControlsBlock' );
+const BlockControlsInline = createSlotFill( 'BlockFormatControls' );
+const BlockControlsOther = createSlotFill( 'BlockControlsOther' );
```

Comment:
The naming seems to be a bit inconsistent/confusing here.

Reply:
Two of them are for backward compatibility but yes `BlockFormatControlsBlock` this one is wrong yes.